docs/USAGE-GUIDE.md

```diff
@@ -1777,6 +1777,164 @@ handle. The handle is considered invalid after **s2n_connection_free** is used.
 [s2n_connection_wipe](#s2n\_connection\_wipe) does not need to be called prior to this function. **s2n_connection_free** performs its own wipe
 of sensitive data.
 
+## TLS1.3 Pre-Shared Key (PSK) Related Calls
+
+S2N-TLS provides support for pre-shared keys. PSKs can be established out of band (external PSKs) or in a previous connection (resumption PSKs).
```

Comment:
```suggestion
S2N-TLS provides support for pre-shared keys (PSKs). PSKs can be established out of band (external PSKs) or in a previous connection (resumption PSKs).
```

Reply:
It is already abbreviated on line 1780.

Reply:
If you feel the need to write out "pre-shared keys", it's probably worth repeating the abbreviation too. I agree with writing it out the first time. I'm not sure I would actually even including the abbreviation in the header.